### PR TITLE
Migrate documentation from Wiki

### PR DESCRIPTION
**Pull request by dduportal**
Hello 👋 

This PR introduces the following changes:

- Migrate documentation from wiki to Github, in the file `README.md`, as per https://www.jenkins.io/blog/2019/10/21/plugin-docs-on-github/
- Change plugin URL in update center to specify this README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,11 +3,758 @@
 [![Build
 Status](https://ci.jenkins.io/job/Plugins/job/cobertura-plugin/job/master/badge/icon)](https://ci.jenkins.io/job/Plugins/job/cobertura-plugin/job/master/)
 
+[![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/cobertura.svg)](https://plugins.jenkins.io/cobertura)
+
+<!-- Commenting out the changelog badge as >
+<!-- [![GitHub release](https://img.shields.io/github/release/jenkinsci/cobertura.svg?label=changelog)](https://github.com/jenkinsci/cobertura-plugin/releases/latest) -->
```
Comment (jeffpearce) [inline README.md:9]: Hi @dduportal. Thanks for the PR! I'm curious about this commented out code - could we just remove it?

File: README.md [hunk 2]
```diff
@@ -3,11 +3,758 @@
...
+[Cobertura](http://cobertura.sourceforge.net/). Jenkins will generate
+the trend report of coverage.The Cobertura plugin can be [downloaded
 here](http://updates.jenkins-ci.org/download/plugins/cobertura/).
 
-For more information, see [the Wiki
-page](https://wiki.jenkins-ci.org/display/JENKINS/Cobertura+Plugin).
+## Version History
+
+- Version history can be found below: [CHANGELOG](#changelog)
+- Tagged releases can be found on the following page: [Releases](https://github.com/dduportal/cobertura-plugin/releases)
```
Comment (jeffpearce) [inline README.md:21]: https://github.com/jenkinsci/cobertura-plugin/releases

Comment (dduportal) [reply README.md]: Good catch, I'm sorry for the bad copy and paste

File: README.md (hunk 1 above)
Comment (dduportal) [reply README.md]: Removed it! I forgot to do it when pushing.

**Comment by dduportal** [comment]
Hi @jeffpearce thanks for the careful review!
Your feedbacks had been taken in account.

However, bout the release/changelog, as you can see I'v added the latest releases, but without any content (as I did not know what to put in).

Is there any plans to use the new plugin release system (with changes in the GH releases text payload)?

**Comment by dduportal** [comment]
@jeffpearce thanks for the review and the merge!
To be honest I'm not sure where to look at, let's ask in the gitter channel :)

**Comment by dduportal** [comment]
Hi @jeffpearce , we have an anwser:

- https://www.jenkins.io/doc/developer/publishing/documentation/#changelogs
- https://www.youtube.com/watch?v=PaQsvli92XY
and
- https://www.youtube.com/watch?v=lphs-7s4NtQ

**Pull request merged**